build/common.sh

```diff
@@ -45,7 +45,7 @@ readonly KUBE_GCS_DELETE_EXISTING="${KUBE_GCS_DELETE_EXISTING:-n}"
 
 # Constants
 readonly KUBE_BUILD_IMAGE_REPO=kube-build
-readonly KUBE_BUILD_IMAGE_CROSS_TAG="v1.6.2-2"
+readonly KUBE_BUILD_IMAGE_CROSS_TAG="$(cat build/build-image/cross/VERSION)"
```

Comment:
We've debated about this earlier, but I think I'm fine with having this.
The pro is no manual update here again, the con is that we must always have the stable kube-cross image in here. But that don't bother me much.